src/services/exprensesService.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+const { getAll } = require('../services/usersServer');
```

Comment:
Consider renaming the imported function 'getAll' from 'usersServer' to a more descriptive name, like 'getAllUsers', to make the code more readable.